packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java

```diff
@@ -61,6 +61,7 @@ public class TextAttributeProps {
   public static final short TA_KEY_LINE_BREAK_STRATEGY = 25;
   public static final short TA_KEY_ROLE = 26;
   public static final short TA_KEY_TEXT_TRANSFORM = 27;
+  public static final short TA_KEY_MAX_FONT_SIZE_MULTIPLIER = 29;
```

Comment:
Note that I've skipped 28 here. It's reserved by `TA_KEY_ALIGNMENT_VERTICAL` in `attributedstring/conversions.h`, but is missing its equivalent in this file. Might be another bug.